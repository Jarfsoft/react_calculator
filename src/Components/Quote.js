import React from 'react';

import './Quote.css';

const Quote = () => (
  <section className="quote">
    <h3>Quote of the day</h3>
    <strong>
      <p>Mathematics is not about number, equations, computations or algorithms:</p>
      <p>is about understading.</p>
    </strong>
    <span> - William Paul Thurston.</span>
  </section>
);

export default Quote;
