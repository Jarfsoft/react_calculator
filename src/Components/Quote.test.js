import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quote';

it('render quote page', () => {
  const quote = renderer
    .create(<Quote />)
    .toJSON();
  expect(quote).toMatchSnapshot();
});
