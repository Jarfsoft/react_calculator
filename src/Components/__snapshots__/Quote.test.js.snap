// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render quote page 1`] = `
<section
  className="quote"
>
  <h3>
    Quote of the day
  </h3>
  <strong>
    <p>
      Mathematics is not about number, equations, computations or algorithms:
    </p>
    <p>
      is about understading.
    </p>
  </strong>
  <span>
     - William Paul Thurston.
  </span>
</section>
`;
